hack/ci/windows.ps1

```diff
@@ -132,6 +132,12 @@ $FinallyColour="Cyan"
 #$env:SKIP_COPY_GO="yes"
 #$env:INTEGRATION_TESTFLAGS="-test.v"
 
+# Use containerd runtime on Windows Server 2022 preview and later
+if ([System.Environment]::OSVersion.Version.Build -ge 20295) {
+    Write-Host -ForegroundColor Green "INFO: Running the daemon under test in ContainerD mode"
+    $env:DOCKER_WINDOWS_CONTAINERD_RUNTIME=1
+}
+
```

Comment:
Make sense. Let's merge #42528 then first and I will update this PR after that.

Reply:
@thaJeztah or well. I start my summer vacation today and will be out of keyboard couple of weeks so maybe it will make sense that you pull all other commits expect default runtime change https://github.com/moby/moby/pull/41479/commits/fd880bdd062efdb6353218fc75bd181f692ec9e5 to it if you are working on meantime.

Also note that libnetwork merge to moby/moby added one new test which does not works on Windows + ContainerD combination which most probably need to be skipped to get that PR merged:
```
[2021-06-14T19:25:57.216Z] === Failed
[2021-06-14T19:25:57.216Z] === FAIL: github.com/docker/docker/libnetwork/networkdb TestNetworkDBCRUDMediumCluster (9.52s)
[2021-06-14T19:25:57.216Z] 2021/06/14 19:24:05 Closing DB instances...
[2021-06-14T19:25:57.216Z] time="2021-06-14T19:24:05Z" level=error msg="node: 9b625bc495af is unknown to memberlist"
[2021-06-14T19:25:57.216Z]     networkdb_test.go:439: Network existence verification failed
```

Reply:
This makes sense to me. Better to land the work for containerd and skip the default changing, so we can have it up on CI and iterate on the rest, than have the whole stack stuck in a PR.